README.md

```diff
@@ -1,2 +1,83 @@
-# module-ballerinax-financial.swiftmt
-SWIFT MT message format library
+# Ballerina SWIFT MT Library
+
+## Overview
+
+SWIFT MT (Message Type) messages are a set of standardized financial messages used globally in interbank communication, allowing secure and efficient cross-border transactions. These messages follow a specific structure to ensure consistency in financial data exchange, supporting various operations such as payments, securities trading, and treasury transactions.
+
+The `ballerinax/swiftmt` library provides support for creating, parsing, and manipulating SWIFT MT messages in Ballerina. It facilitates the handling of commonly used MT message types, enabling developers to work with structured financial data in a flexible and programmatic manner. This library is particularly useful for building financial messaging systems that integrate SWIFT MT standards. 
+
+## Supported Message Types
+
+- SWIFT MT 1XX Category
+- SWIFT MT 2XX Category
+- SWIFT MT 9XX Category
+- SWIFT MT nXX Category
+
+## Usage
+
+### Converting SWIFT fin message to Record value
+
+```ballerina
+import ballerina/io;
+import ajai21/swiftmt;
+
+public function main() returns error? {
+    string finMessage = "{1:F01CHASUS33AXXX0000000000}{2:I900CRESCHZZXXXXN}{4:\n" +
+                ":20:C11126A1378\n" +
+                ":21:5482ABC\n" +
+                ":25:9-9876543\n" +
+                ":32A:090123USD233530,\n" +
+                "-}"; 
+    record {|anydata...;|} customizedMessage = check swiftmt:swiftMtParser(finMessage);
```

Comment:
The method name doesn't meet naming standards.

It should be something like `parse`.

Reply:
You can use `record {}` instead of `record {|anydata...;|}`.

Reply:
fixed in `99b8222`